styles/index.css

```diff
@@ -0,0 +1,640 @@
+  font-size: 14px;
+}
+
+.box-items-left p:hover {
+  text-decoration: none;
+}
+
+.article4-bottom-text a:hover {
+  text-decoration: underline;
+}
+
+.photogallery-individual-box p:nth-child(3) {
+  line-height: 22px;
+  margin-bottom: 0;
+  color: #121212;
+}
+
+.footer-bottom a:hover,
+.footer-title a:hover,
+.footer-main-column a:hover {
+  text-decoration: underline;
+}
+
+.photogallery-sideVar ul li a:hover,
+.photogallery-individual-box p:nth-child(3):hover {
+  color: gray;
+  cursor: pointer;
+}
+
+.space {
+  font-weight: 600;
+  opacity: 0.8;
+}
+
+.space:hover {
+  text-decoration: underline;
+}
+
+.right-navBar {
+  align-items: baseline;
+}
+
+.right-navBar button {
+  background-color: #567b95;
+  border: 1px solid #326891;
+  border-radius: 2px;
+  height: 32px;
+  color: white;
+  margin-left: 10px;
+  font-weight: 700;
+}
+
+.mainImage {
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 80px;
+  width: 1000px;
+}
+
+.mainImage img {
+  width: 1000px;
+  height: auto;
+}
+
+.mainImage h1 {
+  font-size: 35px;
+  font-weight: 600;
+  padding-top: 20px;
+  padding-left: 150px;
+  padding-right: 300px;
+  padding-bottom: 20px;
+  line-height: 45px;
+  font-style: italic;
+}
+
+.box-items-left span {
+  color: #333;
+  font-size: 16px;
+}
+
+.article4-img-container span {
+  opacity: 0.5;
+  font-size: 14px;
+}
+
+.article3-image-grid span {
+  font-size: 12px;
+  font-weight: bolder;
+}
+
+.mainImage p span {
+  font-size: 12px;
+  opacity: 0.7;
+}
+
+.article {
+  width: 600px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.main-box-items {
+  display: flex;
+  margin-top: 50px;
+  margin-bottom: 50px;
+}
+
+.box-items-left {
+  width: 300px;
+}
+
+.box-items-left h3 {
+  color: #333;
+  font-weight: 600;
+  text-decoration: underline;
+}
+
+.box-items-right {
+  width: 300px;
+  margin-top: 45px;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+}
+
+.box-items-right a :hover {
+  opacity: 0.4;
+}
+
+.article2-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 900px;
+  margin-left: calc((100% - 600px) / 2);
+  margin-right: auto;
+}
+
+.article2 {
+  width: 600px;
+}
+
+.aside-container {
+  width: 210px;
+  display: flex;
+  flex-direction: column;
+  padding-top: 950px;
+}
+
+.aside-container > h3 {
+  font-size: 14px;
+  font-weight: 700;
+  padding-bottom: 12px;
+  margin-bottom: 15px;
+  border-bottom: 1px solid gray;
+}
+
+.aside-container-box {
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 15px;
+}
+
+.aside-container-box img {
+  width: 75px;
+  height: 75px;
+}
+
+.aside-container-box h3 {
+  font-size: 14px;
+  padding-left: 10px;
+  padding-top: 5px;
+  line-height: 15px;
+}
+
+.article3-header h3 {
+  text-align: left;
+  font-weight: 700;
+}
+
+.footer-main-column h3 {
+  font-size: 13px;
+  font-weight: 700;
+  color: rgb(51, 51, 51);
+}
+
+.aside-container-box h3:hover {
+  text-decoration: underline;
+}
```

Comment:
This selector has the same style definition of selectors on lines 169 and 196. Group selectors to minimize your code.